dashboard/test/lib/expired_deleted_account_purger_test.rb

```diff
@@ -288,7 +289,18 @@ def teardown
       deleted_before: 2.days.ago,
       dry_run: true
 
-    NewRelic::Agent.expects(:record_metric).never
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/SoftDeletedAccounts", is_a(Integer))
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/AccountsPurged", 0)
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/AccountsQueued", 0)
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/ManualReviewQueueDepth", is_a(Integer))
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/DryRunAccountsPurged", 2)
+    NewRelic::Agent.stubs(:record_metric).
+      once.with("Custom/DeletedAccountPurger/DryRunAccountsQueued", 0)
```

Comment:
If we want to ensure that these were called, use `expects` instead of `stubs`. If we just want to stub it, we don't need the `once`. Actually we don't _need_ `once` with `expects` either because it's the default:

From http://gofreerange.com/mocha/docs/Mocha/Expectation.html#once-instance_method
> Note that this is the default behaviour for an expectation, but you may wish to use it for clarity/emphasis.

Reply:
:man_facepalming: I think they work anyway, because of the `once` modifier.  I'll go through and clean this up throughout in a followup PR right now.